x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/components/protection_radio.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import { useDispatch } from 'react-redux';
+import { cloneDeep } from 'lodash';
+import { htmlIdGenerator, EuiRadio } from '@elastic/eui';
+import {
+  ImmutableArray,
+  ProtectionModes,
+  UIPolicyConfig,
+} from '../../../../../../../common/endpoint/types';
+import { MacPolicyProtection, PolicyProtection } from '../../../types';
+import { usePolicyDetailsSelector } from '../../policy_hooks';
+import { policyConfig } from '../../../store/policy_details/selectors';
+import { AppAction } from '../../../../../../common/store/actions';
+import { useLicense } from '../../../../../../common/hooks/use_license';
+
+export const ProtectionRadio = React.memo(
+  ({
+    protection,
+    protectionMode,
+    oses,
+    label,
+  }: {
+    protection: PolicyProtection;
+    protectionMode: ProtectionModes;
+    oses: ImmutableArray<Partial<keyof UIPolicyConfig>>;
```

Comment:
I don't know why, but I keep reading this as `oses` and not `OSes`. maybe rename to `osList`